lib/tomo/templates/config.rb.erb

```diff
@@ -3,6 +3,7 @@ plugin "env"
 plugin "bundler"
 plugin "rails"
 plugin "nvm"
+# plugin "nodenv"
```

Comment:
Let's completely replace `nvm` with `nodenv`. 🙂 Can you update the template to replace all occurrences and remove the commented lines?